src/components/teaser/teaser.ts

```diff
@@ -81,14 +92,32 @@ export class SbbTeaser extends LitElement implements LinkProperties {
 
     setAttributes(this, hostAttributes);
 
+    const style: Readonly<StyleInfo> = this.userDefinedAspectRatio
+      ? { '--sbb-teaser-image-user-aspect-ratio': `${this.userDefinedAspectRatio}` }
+      : {};
+
     /* eslint-disable lit/binding-positions */
     return html`
       <${unsafeStatic(TAG_NAME)} class="sbb-teaser" ${spread(attributes)}>
         <span class="sbb-teaser__container">
           <span class="sbb-teaser__image-wrapper">
-            <slot name="image"></slot>
+            <slot name="image"
+                  style=${styleMap({
+                    ...style,
+                    '--sbb-teaser-image-user-width': `${this.userDefinedWidth}px`,
+                  })}
+            ></slot>
```

Comment:
Not a fan of the style on the slot tag 😞 
I would rather have it on the `host` (and we could also argue that `user-defined-aspect-ratio` and `user-defined-width` are redundant since the consumer could override `--sbb-teaser-image-user-aspect-ratio` and `--sbb-teaser-image-user-width` to reach the same result)

Reply:
Removed